cosmetics-web/app/models/user.rb

```diff
@@ -15,6 +15,8 @@ class User < ApplicationRecord
   validates :new_email, email: { message: :invalid, allow_nil: true }
   validates :new_email, uniqueness: true, allow_nil: true
   validate  :new_email_not_registered
+  validates :password,
+            common_password: { message: I18n.t(:too_common, scope: %i[activerecord errors models user attributes password]) }
```

Comment:
I18n can infer the scope and we can turn the validation message into simply `:too_common` 
```suggestion
  validates :password, common_password: { message: :too_common }
```